helix-core/pom.xml

```diff
@@ -159,6 +159,11 @@ under the License.
       <artifactId>metrics-core</artifactId>
       <version>3.2.3</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <version>4.5.8</version>
```

Comment:
nit, can you have a try to remove the corresponding entry in the helix-rest pom? I think that would be unnecessary with this.